lib/pure/collections/tables.nim

```diff
@@ -240,7 +240,7 @@ type
     ## <#newTable,int>`_.
 
 const
-  defaultInitialSize* = 64
+  tableDefaultInitialCapacity* {.intdefine.} = 64
```

Comment:
ditto

Reply:
Just in case someone wants to control them individually for some reason.